index.js

```diff
@@ -143,16 +205,52 @@ function testGenOperation(swagger, path, operation, config) {
  * @returns {string|Array} set of all tests for a path
  */
 function testGenPath(swagger, path, config) {
-  var operations = swagger.paths[path];
+  var parameters = {
+    queryParameters: [],
+    bodyParameters: [],
+    headerParameters: [],
+    formParameters: [],
+    pathParameters: []
+  };
+  var childProperty = swagger.paths[path];
   var result = [];
-  var op;
+  var property;
+  var param;
   var validOps = ['get', 'put', 'post', 'delete', 'options', 'head', 'patch'];
+  var type;
 
-  noResponseDefinitions = true;
+  if (childProperty.hasOwnProperty('parameters')) {
+    // only adds body parameters to request, ignores query params
+    for (param in childProperty.parameters) {
+      if (parameters.hasOwnProperty(param)) {
+        type = childProperty.parameters[param];
+        switch (type.in) {
+          case 'query':
```

Comment:
there is also a `formData` case. See the [spec](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#parameterObject)